validator/sawtooth_validator/state/state_view.py

```diff
@@ -32,12 +32,20 @@ def __init__(self, database):
         """
         self._database = database
 
-    def create_view(self, state_root_hash):
+    def create_view(self, state_root_hash=None):
         """Creates a StateView for the given state root hash.
 
+        Args:
+            state_root_hash (str): The state root hash of the state view
+                to return.  If None, returns the state view for the
         Returns:
             StateView: state view locked to the given root hash.
         """
+        # If we are not given a state root hash, then return the default Merkle
+        # database
+        if state_root_hash is None:
```

Comment:
This could be simplifed to:
```
db = MerkleDatabase(self._database)
if state_root_hash is not None:
    db.set_root_hash(state_root_hash)

return Stateview(db)
```
I might have the wrong function name above, but I think it should work.